docs/kb/error-codes-messages.md

```diff
@@ -547,7 +547,7 @@ names. Also check that the request has been properly formatted.
 ```
 
 **Problem**: Your Pardot account has a SalesForce
-connector. When a SalesForce connector is present, opportunities
```

Comment:
Cool, so prospect accounts where the only other locked asset you found in your research?  In the ticket that was just an example of one but there could be more.

Reply:
I tested all of the calls on postman and they all looked okay.

Reply:
I mainly followed this: https://salesforce.quip.com/jpWyA5I8TgXw